packages/office-ui-fabric-react/src/components/ContextualMenu/examples/ContextualMenu.CustomMenuList.Example.tsx

```diff
@@ -2,11 +2,78 @@ import * as React from 'react';
 import { DefaultButton } from 'office-ui-fabric-react/lib/Button';
 import { SearchBox } from 'office-ui-fabric-react/lib/SearchBox';
 import { Icon } from 'office-ui-fabric-react/lib/Icon';
-import './ContextualMenuExample.scss';
 import { IContextualMenuListProps, IContextualMenuItem } from 'office-ui-fabric-react/lib/ContextualMenu';
 import { IRenderFunction } from 'office-ui-fabric-react/lib/Utilities';
 
-const ITEMS: IContextualMenuItem[] = [
+export const ContextualMenuWithCustomMenuListExample: React.FunctionComponent = () => {
+  const [items, setItems] = React.useState(menuItems);
+
+  const onAbort = () => {
```

Comment:
Should probably wrap `onAbort` and `onChange` in `useConstCallback` from `@uifabric/react-hooks` to prevent the callback from mutating.